<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>ADD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSub">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>701</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent; border: none; color: white; font-size: 20px; font-weight: bold; </string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Backward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Play_Pause">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Seek_Forward">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Volume">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volume">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>601</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_Time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_video_File">
    <property name="title">
     <string>Select video File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionenglish"/>
    <addaction name="actionrussian"/>
   </widget>
   <addaction name="menuSelect_video_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionenglish">
   <property name="text">
    <string>english</string>
   </property>
  </action>
  <action name="actionrussian">
   <property name="text">
    <string>russian</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
